File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.c
int removeDuplicates(int* nums, int numsSize) {
    if(numsSize==0){
        return 0;
    }
    int count=0;
    for(int i=0;i<numsSize;i++){
        if(i<numsSize-1 && nums[i]==nums[i+1]){
            continue;
        }
        else{
            nums[count]=nums[i];
            count++;
        }
    }
    return count;
    
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums.length ==0 ){
            return 0;
        }
       int rd=1;
        for(int i=1;i<nums.length;i++){
            if(nums[rd-1] != nums[i]){
                nums[rd]=nums[i];
                rd++;
            }
        }
        return rd;
    }
}